pkgs/build-support/fetchgithub/default.nix

```diff
@@ -30,8 +30,10 @@ let
   };
   fetcherArgs = (if useFetchGit
     then {
-      inherit rev deepClone fetchSubmodules; url = "${baseUrl}.git";
+      inherit deepClone fetchSubmodules; url = "${baseUrl}.git";
     } // lib.optionalAttrs (leaveDotGit != null) { inherit leaveDotGit; }
+      # rev is not compulsory for fetchGit.
```

Comment:
```suggestion
```
I don't think we need that. It is obvious that rev can be omitted if it can be nil.